app/views/skins/confirm_delete.html.erb

```diff
@@ -0,0 +1,13 @@
+<!--Descriptive page name, messages and instructions-->
+<h2 class="heading"><%= ts("Delete Skin") %></h2>
+
+<!--main content-->
+<%= form_for(@skin, :html => { method: :delete, class: "simple destroy" }) do |f| %>
+  <p class="caution notice">
+    <%= ts("Are you sure you want to <strong><em>delete</em></strong> the skin \"%{skin_title}\"?", skin_title: @skin.title).html_safe %>
```

Comment:
Brakeman/Hakiri reported:

https://hakiri.io/github/otwcode/otwarchive/master/4ebb1ec11cd4ba52c941241b7959887d86f8a478/warnings/7afdefaa9a88ca

because I think we're trusting HTML in the prompt and in the skin title. So we should not escape the HTML in the message, but still escape any HTML in the skin's title. Can you set up [safe HTML translations](https://guides.rubyonrails.org/i18n.html#using-safe-html-translations) instead?